pouchdb-core/pouchdb-core.d.ts

```diff
@@ -13,9 +16,9 @@ declare namespace PouchDB {
         type AnyCallback = Callback<any, any>;
         type DocumentId = string;
         type DocumentKey = string;
+        type AttachmentId = string;
         type RevisionId = string;
         type Availability = 'available' | 'compacted' | 'not compacted' | 'missing';
-        type Attachment = string | ArrayBuffer;
```

Comment:
Ah, right.  So I would keep this type but change it to `ArrayBuffer` to `string | Blob | Buffer`.